### PR TITLE
Fix Discord presence crashing on Campaign

### PR DESCRIPTION
**Pull request by Zanieon**
Small oversight by my part when i did the feature improvements to the Discord RPC plugin, this should fix the issue when playing Campaign or Coop.

EDIT: I took the liberty to also use the PR to fix some other minor issues in regards to report how many players are currently playing the server and the actual max playerslots the server have in total.

### CODE REVIEW

**Review by GeckoEidechse — APPROVED** [review]
Confirmed working in testing that it no longer causes a script compiliation error when running the campaign compared to without the PR.

File: Northstar.Client/mod/scripts/vscripts/presence/cl_presence.nut [hunk 1]
```diff
@@ -41,6 +48,9 @@ GameStateStruct function DiscordRPC_GenerateGameState( GameStateStruct gs )
         else
             gs.fd_waveNumber = -1 // Tells plugin it's on Wave Break
     }
+	#else
+	gs.fd_waveNumber = -1 // Unecessary for campaign so return -1
+	#endif
```
Comment (GeckoEidechse) [inline Northstar.Client/mod/scripts/vscripts/presence/cl_presence.nut:53]: Weird indentation here comes from the fact that the file mixes tabs and spaces :skull:
(Yet another reason we could finally use a proper Squirrel formatter)

**Review by GeckoEidechse — APPROVED** [review]
Not super happy with multiple changes in one file but I also want this fixes so I'm letting it slide...

Otherwise changes make sense.

Also just noticed that #869 introduced mixed whitespace indentation but kinda too late now. Oh well.

File: Northstar.Client/mod/scripts/vscripts/presence/cl_presence.nut (hunk 1 above)
Comment (F1F7Y) [reply Northstar.Client/mod/scripts/vscripts/presence/cl_presence.nut:53]: .editorconfig <3

Comment (GeckoEidechse) [reply Northstar.Client/mod/scripts/vscripts/presence/cl_presence.nut:53]: Even then we still need to fix up existing files... :,)

**Pull request merged**